src/SFA.DAS.ApprenticeCommitments.Web.AcceptanceTests/Features/ConfirmIdentitySteps.cs

```diff
@@ -37,9 +38,18 @@ public void GivenTheApprenticeHasNotLoggedIn()
         [Given("the apprentice has logged in")]
         public void GivenTheApprenticeHasLoggedIn()
         {
-            TestAuthenticationHandler.AddUser(_registrationId);
+            TestAuthenticationHandler.AddUser(_userContext.RegistrationId);
             _context.Web.Client.DefaultRequestHeaders.Authorization =
-                new AuthenticationHeaderValue(_registrationId.ToString());
+                new AuthenticationHeaderValue(_userContext.RegistrationId.ToString());
+
+            _context.OuterApi.MockServer.Given(
+               Request.Create()
+                   .UsingGet()
+                   .WithPath($"/apprentices/{_userContext.RegistrationId}/currentapprenticeship")
```

Comment:
Didn't think we were using a currentapprenticeship concept. Should it be removed

Reply:
You're right, this is a remnant from the previous code that wasn't changed.  It has been now.